src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartTaskDialogAction.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package com.redhat.devtools.intellij.tektoncd.actions.component;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.redhat.devtools.intellij.common.utils.ExecHelper;
+import com.redhat.devtools.intellij.common.utils.UIHelper;
+import com.redhat.devtools.intellij.tektoncd.actions.TektonAction;
+import com.redhat.devtools.intellij.tektoncd.tkn.Resource;
+import com.redhat.devtools.intellij.tektoncd.tkn.Tkn;
+import com.redhat.devtools.intellij.tektoncd.tree.TaskNode;
+import com.redhat.devtools.intellij.tektoncd.ui.component.StartTaskDialog;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.swing.tree.TreePath;
+import java.io.IOException;
+import java.util.List;
+
+public class StartTaskDialogAction extends TektonAction {
```

Comment:
To be unifom, should be renamed as StartTaskAction

Reply:
My idea was to create a new action and call it StartTaskAction. Right now the user can only use the dialog to set up the inputs/outputs but what if someone prefer to write everything in the editor? So i had in mind to add a new button which allows people to open the TaskRun on the editor and let them edit it. When it's finished they could "right-click" the file and select run task. Another point is if we follow what's happening in vscode. They are working to add two more nodes to display taskRun and pipelineRun separately. In that case we could use the run task action to just start the task with the inputs/outputs found in the taskrun. WDYT?

Reply:
Regarding the editor, you are messing 2 concerns: starting a task and updating it. Look at Jenkins there are 2 differents actions for a job (pipeline) start a build or edit the pipeline. And even if there are extra node, the concern is always to start a task to I think we should remove the Dialog from the class name to be inline with other IJ plugins